src/LinAlg/hiopMatrixRajaSparseTriplet.cpp

```diff
@@ -75,6 +75,13 @@
 #include <cassert>
 // #include <numeric> //std::inclusive_scan is only available after C++17
 
+#define MY_RAJA_GPU_BLOCK_SIZE 128
+
+using my_raja_exec   = RAJA::cuda_exec<MY_RAJA_GPU_BLOCK_SIZE>;
+
+#define RAJA_LAMBDA [=] __device__
```

Comment:
I don't see this execution policy used anywhere in this PR, so I'm a bit confused about why it's here. The block size, exec policy, and the `RAJA_LAMBDA` macro should already be defined elsewhere (`hiop_raja_defs.hpp`?).